### ISSUE TITLE
large amount of packet receive errors

### ISSUE THREAD

**Issue by aeyakovenko** [solana-labs/solana#1091]
```
anatolyyakovenko@master-perf-testnet-solana-com:~$ netstat -suna
IcmpMsg:
    InType0: 1374
    InType3: 4486
    InType8: 42
    InType11: 71
    OutType0: 42
    OutType3: 5092
    OutType8: 1377
    OutType11: 2199
Udp:
    4487413003 packets received
    8315 packets to unknown port received.
    4238702717 packet receive errors
    15306820 packets sent
    RcvbufErrors: 4238702717
    IgnoredMulti: 6
UdpLite:
IpExt:
    InMcastPkts: 16
    OutMcastPkts: 20
    InBcastPkts: 6
    OutBcastPkts: 6
    InOctets: 2027416793841
    OutOctets: 814939422518
    InMcastOctets: 2576
    OutMcastOctets: 2736
    InBcastOctets: 284
    OutBcastOctets: 284
    InNoECTPkts: 8728358653
    InECT0Pkts: 11
```

it looks like the majority of the packets are dropped.  I am not sure why.  maybe we need access to the raw NIC on google cloud instead of whatever virtualization its using.

**Comment by aeyakovenko** [comment]
@mvines would it be worthwhile to use something like telegraph to dump network stats into influx?  or should we just poll on our own.

https://lkhill.com/telegraf-influx-grafana-network-stats/

I am not familiar enough with it

**Comment by mvines** [comment]
We could potentially use telegraph, but dumping metrics into influx is so easy I’d suggest we just roll our own monitoring (similar to the oom monitoring perhaps).  If you already have a list of information sources in mind i can glue it up quick

**Comment by aeyakovenko** [comment]
@mvines     these two are immediately useful
```
4238702717 packet receive errors
4487413003 packets received
```

**Comment by garious** [comment]
Does this change with @pgarg66's change to use multiple recv sockets?

**Comment by aeyakovenko** [comment]
@garious it would be great to see the change in influx.  i am not sure yet

**Comment by aeyakovenko** [comment]
Fixed in 0.10
